mirgecom/artificial_viscosity.py

```diff
@@ -158,9 +160,11 @@ class _AVRTag:
 
 
 def av_laplacian_operator(discr, boundaries, fluid_state, alpha, gas_model=None,
-                          kappa=1., s0=-6., time=0, operator_states_quad=None,
-                          grad_cv=None, quadrature_tag=None, boundary_kwargs=None,
+                          kappa=1., s0=-6., time=0, quadrature_tag=DISCR_TAG_BASE,
+                          volume_dd=DD_VOLUME_ALL, boundary_kwargs=None,
```

Comment:
Delete `boundary_kwargs`?

Reply:
Can't yet, some of the drivers still use it (isolator and nozzle).